pkg/devfile/validate/components.go

```diff
@@ -2,13 +2,14 @@ package validate
 
 import (
 	"fmt"
+
 	"github.com/openshift/odo/pkg/devfile/parser/data/common"
 )
 
 // Errors
 var (
-	ErrorNoComponents           = "no components present"
-	ErrorNoDockerImageComponent = fmt.Sprintf("odo requires atleast one component of type '%s' in devfile", common.DevfileComponentTypeDockerimage)
+	ErrorNoComponents         = "no components present"
+	ErrorNoContainerComponent = fmt.Sprintf("odo requires atleast one component of type '%s' in devfile", common.ContainerComponentType)
 )
 
 // ValidateComponents validates all the devfile components
```

Comment:
can we make this function lowercase? it seems to not be used anywhere outside this package.